config/kubernetes/development/cronjob-close-rejected-offender-sar.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: batch/v1
```

Comment:
Currently this cronjob is set to run in the development namespace but nowhere else.

Reply:
I have only created this in the kubernettes\development I will copy the file into the other kubernetes environments once working.  Is that ok?